docs-src/ref-programs-step.scrbl

```diff
@@ -247,7 +247,7 @@ fork()
 }
 
 where:
-@reachin{TOKENS_EXPR} is an expression that evalues to a tuple of @reachin{Token}s.
+@reachin{TOKENS_EXPR} is an expression that evaluates to a tuple of @reachin{Token}s.
```

Comment:
noticed this while reading through the docs

Reply:
Best to submit these separately for easy merging :) or just push 'em when you have the power